src/components/strategies/overview/strategyBlock/StrategyBlockManage.tsx

```diff
@@ -48,6 +48,11 @@ export const StrategyBlockManage: FC<Props> = (props) => {
   const order1 = useOrder(strategy.order1);
   const { type, slug } = useParams({ from: '/explore/$type/$slug' });
 
+  const { user } = useWeb3();
```

Comment:
I'd suggest wrapping it in a custom hook, i.e. `useIsStrategyOwner(id)` - not only because of the usual reasons but also because it will be a single place to replace the implementation if we eventually move to rely on the backend for this